### DISCUSSION TITLE
파일처리, 운영체제, 컴파일러 과제 아카이브

### DISCUSSION THREAD

**Discussion by gomjellie** [Soongsil-Developers/.github#8] [필기, 공부 정리, 자료공유]
2, 3년 전에 했던 과제들을 모아놓은 레포입니다 :)
아마 비슷하게 그대로 나오는것도 있을거에요

레포지토리 주소: https://github.com/gomjellie/ssucse

**Comment by sang-w0o** [comment]
오예~~~~~
